PrepInsta Top 100 Codes/Armstrong_Number.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+
+public class Armstrong_Number {
+     public static void main(String[] args) {
+        // first we have to give a input number using the Scanner class also we have to import the package of Scanner class
+        Scanner sc=new Scanner(System.in);      
+        System.out.println("Enter a number");
+        int n=sc.nextInt();
+        // we have to store the value of 'n' in a variable 'temp' because with every iteration the value of 'n' will be change.
+        int temp=n; 
+        // we have to declare the a variable 'sum' in which we will store the sum of the cube of each digit of 'n'
+        int sum=0;
+        // we have to declare the variable 'rem' in which we will store the remainder of the division of 'n' by 10
+        int rem;
+        // now we will start our while loop to get the sum of the cube of each digit of 'n'
+        while(n>0)
+        {
+            // we have to store the remainder of the division of 'n' by 10 in the variable 'rem'
+            rem=n%10;
+            // we have to store the cube of 'rem' in the variable 'sum'
+            sum=sum+(rem*rem*rem);
```

Comment:
According to wikipedia Armstrong number is a number that is the sum of its own digits each raised to the power of the number of digits.
But you have applied different logic here.
Yeah it's true that in some website armstrong number logic is exactly same as yours but i prefer to follow the wikipedia so i request you to change this logic.

Reply:
Yeah , now i got your point .